connectors/sources/mysql.py

```diff
@@ -294,6 +300,31 @@ async def _fetchmany_in_batches(self, query):
                 )
 
 
+def row2doc(row, column_names, primary_key_columns, table, timestamp):
+    row = dict(zip(column_names, row))
+    row.update(
+        {
+            "_id": generate_id(table, row, primary_key_columns),
+            "_timestamp": timestamp or iso_utc(),
+            "Table": table,
+        }
+    )
+
+    return row
+
+
+def generate_id(tables, row, primary_key_columns):
+    keys_value = ""
+    for key in primary_key_columns:
+        keys_value += f"{row.get(key)}_" if row.get(key) else ""
```

Comment:
Question: Is it your intention for `keys_value` to always end with an underscore?

Reply:
Nope, just took it from the original implementation, will adjust it, when working on stable/unique ids 👍